src/dispatchers.ts

```diff
@@ -205,6 +272,49 @@ export async function followingCounter(
     return results.length;
 }
 
+type StoredThing = {
+    object: object | string;
+}
+
+export async function inboxDispatcher(
+    ctx: RequestContext<ContextData>,
+    handle: string,
+) {
+    console.log('Inbox Dispatcher');
+    const results = (await ctx.data.db.get<string[]>(['inbox'])) || [];
+    console.log(results);
+    let items: Activity[] = [];
+    for (const result of results) {
+        try {
+            const thing = await ctx.data.globaldb.get<StoredThing>([result]);
+
+            // If the object is a string, it's a URI, so we should to look it up
+            // in the globalDb. If it's not in the globalDb, we should just
```

Comment:
Long term we probably want to fetch it from the network and store it - but this is fine for now

Reply:
Not sure I'm following - In the case of `Announce` we retrieve the announced object and store it at the point of handling the `Announce` activity. So at this point, we already have the object in the db?

(obviously this is only valid for `Announce` activities that have been handled by the handler)